plugins/hydra_optuna_sweeper/example/conf/config.yaml

```diff
@@ -4,7 +4,7 @@ defaults:
 hydra:
   sweeper:
     optuna_config:
-      direction: minimize
+      directions: [minimize]
```

Comment:
This seems to be the only breaking change in the config - for people who are doing single objective training.

How about enabling some backward compatibility by supporting both `direction` and `directions` field. In the plugin, we ignore `direction` if `directions` is setup. Else, we set `directions = [direction]`

Reply:
Thank you for your suggestion. That sounds reasonable. I reverted `OptunaConfig.direction` in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/07bcf4cd6416428792d2851229996217a0e89b98 according to your advice.